sdk/communication/communication-chat/src/models/models.ts

```diff
@@ -60,10 +60,34 @@ export interface ChatMessageContent {
   topic?: string;
   /** Chat message content for messages of types participantAdded or participantRemoved. */
   participants?: ChatParticipant[];
+  /** List of attachments for this message */
+  attachments?: ChatAttachment[];
   /** Identifies a participant in Azure Communication services. A participant is, for example, a phone number or an Azure communication user. This model must be interpreted as a union: Apart from rawId, at most one further property may be set. */
   initiator?: CommunicationIdentifierKind;
 }
 
+/** An attachment in a chat message. */
+export interface ChatAttachment {
+  /** Id of the attachment */
+  id: string;
+  /** The type of attachment. */
+  attachmentType: AttachmentType;
+  /** The type of content of the attachment, if available */
+  contentType?: string;
+  /** The name of the attachment content. */
+  name?: string;
+  /** The URL where the attachment can be downloaded */
+  url: string;
+  /** The URL where the preview of attachment can be downloaded */
+  previewUrl?: string;
+}
+
+export enum KnownAttachmentType {
```

Comment:
don't think these should be here

Reply:
Cleared up offline. We should be keeping this type